mythril/analysis/modules/ether_thief.py

```diff
@@ -8,37 +8,35 @@
 import logging
 
 
-"""
-MODULE DESCRIPTION:
+DESCRIPTION = """
+
+Search for cases where Ether can be withdrawn to a user-specified address. 
 
-Search for cases where Ether can be withdrawn to a user-specified address.
 An issue is reported ONLY IF:
 
 - The transaction sender does not match contract creator;
+- The sender address can be chosen arbitrarily;
+- The receiver address is identical to the sender address;
 - The sender has not previously sent any ETH to the contract account.
 
 This is somewhat coarse and needs to be refined in the future.
 
 """
 
+ARBITRARY_SENDER_ADDRESS = 0xAAAAAAAABBBBBBBBBCCCCCCCDDDDDDDDEEEEEEEE
+
 
 class EtherThief(DetectionModule):
     def __init__(self):
         super().__init__(
             name="Ether Thief",
             swc_id=UNPROTECTED_ETHER_WITHDRAWAL,
             hooks=["CALL"],
-            description=(
-                "Search for cases where Ether can be withdrawn to a user-specified address.\n"
-                "An issue is reported ONLY IF:\n\n"
-                "- The transaction sender does not match contract creator\n"
-                "- The sender has not previously sent any ETH to the contract account.\n\n"
-                "This is somewhat coarse and needs to be refined in the future."
-            ),
+            description=DESCRIPTION,
         )
 
     def execute(self, state_space):
-        logging.debug("Executing module: ETHER_THIEF")
+        logging.debug("Executing module: " + format(self.name))
```

Comment:
+1 on dynamically adding it in. The `logging` factory has a neat twist where it resolves the name lazily - right before writing out. It's best practice to pass in the variable name without inserting it as such
```suggestion
        logging.debug("Executing module: %s", self.name)
```
Sure, it's not a format string, but that just shows how desperately we need a refactoring of the STL. :slightly_smiling_face: